src/test/java/net/serenitybdd/starter/fetests/testCases/BaseTest.java

```diff
@@ -0,0 +1,44 @@
+package net.serenitybdd.starter.fetests.testCases;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import net.serenitybdd.starter.model.User;
+import net.thucydides.core.annotations.Managed;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class BaseTest {
+
+    @Managed
+    WebDriver driver;
+
+    public void startApp() {
+        WebDriverManager.chromedriver().setup();
+        driver = new ChromeDriver();
```

Comment:
driver duplication: the first is managed by Serenity the second here. Please remove. I'm not sure you need WebDriver instance on Test layer